lib/github/api.nit

```diff
@@ -19,7 +19,6 @@
 # For most use-cases you need to use the `GithubAPI` client.
 module api
 
-# TODO to remove
```

Comment:
but but but you add it 14 commits ago https://github.com/nitlang/nit/pull/2765/commits/ae3ec8baee7660fb029a9be55c5a20fca7cc6fc5#diff-910b30e850bf6fd2777c60a2a1627a07R22

Reply:
Yes but it's only used for a hack in the deserialization process using `path`:

https://github.com/nitlang/nit/pull/2765/files#diff-910b30e850bf6fd2777c60a2a1627a07R1134